### PR TITLE
Renovate config for Iac. .

### PR DESCRIPTION
**Pull request by Artlvns**
feat: renovate common config for IaC currently support terraform, tflint and lockfile dependencies.

### CODE REVIEW

File: renovate-config-iac.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Shared Renovate configuration for dsb-norge IaC projects where major updates are skipped",
```
Comment (Laffs2k5) [inline renovate-config-iac.json]: Are majors skipped? The config further down says major.enabled = true 🤔

File: renovate-config-iac.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            "**/_terraform-state/**"
+        ],
+        "pinDigests": true
+    },
+    "commitMessageLowerCase": "never",
+    "prHourlyLimit": 4,
+    "major": {
+        "enabled": true
+    },
+    "ignoreScripts": true,
```
Comment (Laffs2k5) [inline renovate-config-iac.json]: This is applicable only for npm, bun, Composer and Copier. ref. https://docs.renovatebot.com/configuration-options/#ignorescripts

File: renovate-config-iac.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "reviewers": [
+        "dsb-developers"
```
Comment (Laffs2k5) [inline renovate-config-iac.json]: We should not nag all devs with this. Probably need a separate team for IaC devs. Is it possible to hardcode you and me for now?

File: renovate-config-iac.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    ],
+    "timezone": "Europe/Stockholm",
+    "schedule": [
+        "after 01:20am and before 7:30am every weekday",
```
Comment (Laffs2k5) [inline renovate-config-iac.json]: Remember to sync terraform-runner schedule with this interval. No init and plan will run if terraform runners are stopped. We also observed that not all PRs got their workflow executed if many PRs where queued during runner downtime.

File: renovate-config-iac.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        "every weekend"
+    ],
+    "updateNotScheduled": false,
```
Comment (Laffs2k5) [inline renovate-config-iac.json]: My understanding is that this is best set to true for us (which is default). Then renovate will rebase the PR branches outside of the schedule when main receives pushes.

**Review by Laffs2k5 — APPROVED** [review]
Nice work!

Please consider my inputs :-)

File: renovate-config-iac.json
Comment (Laffs2k5) [inline renovate-config-iac.json:1]: Possibly name this to be specific for iac-projects or terraform-projects. We will probably have a different one for terraform-modules.

Comment (Artlvns) [reply renovate-config-iac.json:1]: I suggest leave Iac for now.  If something very special will be for modules we can mirror it to name.  The only things I see for me there is lack of lock files and auto merge for pr's so perhaps *-automerge will follow for modules ?

File: renovate-config-iac.json (hunk 1 above)
Comment (Artlvns) [reply renovate-config-iac.json]: nice catch. forgot to update description.

File: renovate-config-iac.json
Comment (Artlvns) [reply renovate-config-iac.json:1]: or *-tf-modules

File: renovate-config-iac.json (hunk 5 above)
Comment (Artlvns) [reply renovate-config-iac.json]: In my tests each time I pushed something new renovate performed rebase to existing branches.  Let's test out :)

**Pull request merged**